graphql/ui_plugin_build_baron.go

```diff
@@ -127,6 +128,9 @@ func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnIn
 
 	if !ok {
 		projectNotFoundError = true
+		grip.Debug(message.Fields{
+			"message": "Chaya, ui_plugin_build_baron.go, 132",
```

Comment:
I've found it useful to log each field I want to search on separately. Like I'd have "ticket", "function", and "line" fields. It's made searching Splunk easier: I can find all the logging for a particular ticket or for a particular function, for example.